packages/core/src/__tests__/unit/application.unit.ts

```diff
@@ -16,12 +16,14 @@ import {expect} from '@loopback/testlab';
 import {Application, Component, CoreBindings, CoreTags, Server} from '../..';
 
 describe('Application', () => {
+  let app: Application;
+
+  beforeEach(givenApp);
```

Comment:
I am not sure if it's a good idea to always create the same application for all tests. What if we need to call the app constructor with different arguments in a test we will write in a future?

A more robust & future-proof solution is to keep the `app` variable and `afterEach` hooks here (shared by all tests), but keep `beforeEach(givenApp)` inside every `describe` block as it is now.